storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -52,7 +52,7 @@ bool Redo_Log_Reader::find_start_checkpoint_lsn() {
   /* Look for the latest checkpoint */
   Log_checkpoint_location checkpoint;
 
-  if (recv_find_max_checkpoint(*log_sys, checkpoint)) {
+  if (!recv_find_max_checkpoint(*log_sys, checkpoint)) {
```

Comment:
really? the definitions changed to opposite? can you please double check?

Reply:
yes. 
`/** Find the latest checkpoint (check all existing redo log files).
  @param[in,out]  log             redo log
  @param[out]     checkpoint      the latest checkpoint found (if any)
  @return true iff any checkpoint has been found */`